floris/tools/floris_interface.py

```diff
@@ -108,175 +111,134 @@ def copy(self):
-        farm is modeled as if there is no wake in the flow.
-        Yaw angles are used to reduce the power and thrust of
-        the turbine that is yawed.
+        This function is similar to `run()` except that it does not apply a wake model. That is,
+        the wind farm is modeled as if there is no wake in the flow. Yaw angles are used to reduce
+        the power and thrust of the turbine that is yawed.
 
-        Args:
-            yaw_angles (NDArrayFloat | list[float] | None, optional): Turbine yaw angles.
-                Defaults to None.
+        Args: (None)
         """
 
-        if yaw_angles is None:
-            yaw_angles = np.zeros(
-                (
-                    self.floris.flow_field.n_findex,
-                    self.floris.farm.n_turbines,
-                )
-            )
-        self.floris.farm.yaw_angles = yaw_angles
-
-        if power_setpoints is None:
-            power_setpoints = POWER_SETPOINT_DEFAULT * np.ones(
-                (
-                    self.floris.flow_field.n_findex,
-                    self.floris.farm.n_turbines,
-                )
-            )
-        else:
-            power_setpoints = np.array(power_setpoints)
-
-        # Convert any None values to the default power setpoint
-        power_setpoints[
-            power_setpoints == np.full(power_setpoints.shape, None)
-        ] = POWER_SETPOINT_DEFAULT
-        power_setpoints = floris_array_converter(power_setpoints)
-
-        # Check for turbines to disable
-        if disable_turbines is not None:
-
-            # Force to numpy array
-            # disable_turbines = np.array(disable_turbines)
-
-            # Must have first dimension = n_findex
-            if disable_turbines.shape[0] != self.floris.flow_field.n_findex:
-                raise ValueError(
-                    f"disable_turbines has a size of {disable_turbines.shape[0]} "
-                    f"in the 0th dimension, must be equal to "
-                    f"n_findex={self.floris.flow_field.n_findex}"
-                )
-
-            # Must have first dimension = n_turbines
-            if disable_turbines.shape[1] != self.floris.farm.n_turbines:
-                raise ValueError(
-                    f"disable_turbines has a size of {disable_turbines.shape[1]} "
-                    f"in the 1th dimension, must be equal to "
-                    f"n_turbines={self.floris.farm.n_turbines}"
-                )
-
-            # Set power_setpoints and yaw_angles to 0 in all locations where
-            # disable_turbines is True
-            yaw_angles[disable_turbines] = 0.0
-            power_setpoints[disable_turbines] = 0.001 # Not zero to avoid numerical problems
-
-        self.floris.farm.power_setpoints = power_setpoints
-
         # Initialize solution space
         self.floris.initialize_domain()
 
         # Finalize values to user-supplied order
         self.floris.finalize()
 
-    def reinitialize(
+    def _reinitialize(
```

Comment:
Would it make sense for there to be functions like reininitialize() and calculate_wake() which solely exist to prompt users to read the v4 documentation?

Reply:
Yes, that's a nice suggestion. I've added these two functions in https://github.com/NREL/floris/pull/823/commits/058930b053f2065469872bacb9f5ba69c2266b28.